modules/contentbox-admin/views/settings/rawSettingsTable.cfm

```diff
@@ -53,7 +53,7 @@
 					</a>
 					
 					<!--- Delete Command --->
-					<a 	class="btn btn-sm btn-danger confirmIt" 
+					<a 	class="btn btn-sm btn-danger #setting.getIsCore() ? '' : 'confirmIt'#"
```

Comment:
does it make sense to add this @Tropicalista if you already have an if statement below on line 58